tests/regression/test_regression.py

```diff
@@ -68,12 +69,13 @@ def _test_regression(
             with mlflow.start_run(tags=tags, run_name=run_name):
                 overrides = [
                     f"+recipe={test_case.model.task}/{test_case.model.name}",
-                    f"model.otx_model.config.head.num_classes={test_case.dataset.num_classes}",
+                    f"model.otx_model.config.{head_name}.num_classes={test_case.dataset.num_classes}",
                     f"data.data_root={data_root}",
                     f"data.data_format={test_case.dataset.data_format}",
                     f"base.output_dir={test_case.output_dir}",
                     f"seed={seed}",
                     "test=true",
+                    "trainer=gpu",
```

Comment:
Can you add an explicit fixture such as `fxt_accelerator` in https://github.com/openvinotoolkit/training_extensions/pull/2724#discussion_r1424738570? It will make it easily expandable for INTEL devices.

Reply:
That example didn't work for me. Can you explain about proper behavior of fxt_accelerator?

Reply:
Nevermind, I'll do that.